dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/inject/InjectInjectionObjectFactory.java

```diff
@@ -160,6 +161,7 @@ public static Object getObjectInstance(InjectInjectionBinding iBinding, Object t
 
             Object injectionObject = AccessController.doPrivileged(new PrivilegedAction<Object>() {
                 @Override
+                @Trivial //This method is not actually trivial, but to avoid calling toString on a weld proxy as part of trace I will write the trace statements manually
```

Comment:
You've added exit but no entry ... although not sure if it needs either in a doPriv